docs/quickstart.md

```diff
@@ -24,7 +24,7 @@ Open a terminal and verify you have the following:
   * While GCP is also fully supported, we will focus on AWS in this quick start.
 * For sending dummy data to Opstrace, you'll need [Docker](https://docs.docker.com/install) and [Docker Compose](https://docs.docker.com/compose/install)
 
-Note: the code blocks are all copy-and-pastable.
+You will also need an Auth0 account to wire up Google or GitHub log in (sign up for free here).
```

Comment:
Needs a link.

Reply:
Removing this line in favor of the authentication section below.